index.html

```diff
@@ -1,324 +1,404 @@
-              >
-            </dd>
-          </dl>
+            <dl id="document-identifier">
+              <dt>This version</dt>
+              <dd>
+                <a href="https://solid.github.io/webid-profile/" rel="owl:sameAs"
+                  >https://solid.github.io/webid-profile/</a
+                >
+              </dd>
+            </dl>
 
-          <dl id="document-latest-published-version">
-            <dt>Latest published version</dt>
-            <dd>
-              <a
-                href="https://solid.github.io/webid-profile/"
-                rel="rdfs:seeAlso"
-                >https://solid.github.io/webid-profile/</a
-              >
-            </dd>
-          </dl>
+            <dl id="document-latest-published-version">
+              <dt>Latest published version</dt>
+              <dd>
+                <a
+                  href="https://solid.github.io/webid-profile/"
+                  rel="rdfs:seeAlso"
+                  >https://solid.github.io/webid-profile/</a
+                >
+              </dd>
+            </dl>
 
-          <dl id="document-editors">
-            <dt>Editors</dt>
-            <dd id="Virginia-Balseiro">
-              <span about="" rel="schema:creator schema:editor"><span about="https://virginiabalseiro.com/#me"
-                  typeof="schema:Person"><a rel="schema:url" href="https://virginiabalseiro.com/#me"><span
-                      about="https://virginiabalseiro.com/#me" property="schema:name"><span
-                        property="schema:givenName">Virginia</span>
-                      <span property="schema:familyName">Balseiro</span></span></a></span></span>
-            </dd>
+            <dl id="document-editors">
+              <dt>Editors</dt>
+              <dd data-editor-id="129283" id="Virginia-Balseiro">
+                <span about="" rel="schema:creator schema:editor"><span about="https://virginiabalseiro.com/#me"
+                    typeof="schema:Person"><a href="https://virginiabalseiro.com/#me" rel="schema:url"><span
+                        about="https://virginiabalseiro.com/#me" property="schema:name"><span
+                          property="schema:givenName">Virginia</span>
+                        <span property="schema:familyName">Balseiro</span></span></a></span></span>
+              </dd>
 
-            <dd id="Timea-Turdean">
-              <span about="" rel="schema:editor"><span about="https://timea.solidcommunity.net/profile/card#me"
-                  typeof="schema:Person"><a rel="schema:url" href="https://timea.solidcommunity.net/profile/card#me"><span
-                      about="https://timea.solidcommunity.net/profile/card#me" property="schema:name"><span
-                        property="schema:givenName">Timea</span>
-                      <span property="schema:familyName">Turdean</span></span></a></span></span>
-            </dd>
+              <dd id="Timea-Turdean">
```

Comment:
Should @csarven be under Editors as well?

Reply:
Whatever, really, at this point. Added https://github.com/solid/webid-profile/pull/116/commits/01da96607146d705b98cf40a6ca909e054a9f9da